sdk/storage/Azure.Storage.DataMovement/src/Shared/TransferManager.cs

```diff
@@ -260,6 +261,32 @@ public virtual Task<bool> TryPauseTransferAsync(string transferId, CancellationT
             }
         }
 
+        /// <summary>
+        /// Gets the current transfers stored in the <see cref="TransferManager"/>.
+        /// </summary>
+        /// <param name="filterByStatus">
+        /// If specified, the returned list of transfers will have only have the transfers
+        /// of which match the status specified.
+        ///
+        /// If not specified or specified to <see cref="StorageTransferStatus.None"/>,
+        /// all transfers will be returned regardless of status.
+        /// </param>
+        /// <returns></returns>
+        public virtual IEnumerable<DataTransfer> GetTransfers(
+            IEnumerable<StorageTransferStatus> filterByStatus = default)
+        {
+            if (filterByStatus == default)
+            {
+                return _dataTransfers.Select(d => d.Value);
+            }
+            else
+            {
+                return _dataTransfers.
+                    Select(d => d.Value).
+                    Where(x => filterByStatus.Contains(x.TransferStatus)).ToList();
```

Comment:
style nit: Generally I like the approach of putting the `.` a the start of each line of the Linq query. Just feels weird to have the `.` at the end but it's up to you.

```c#
return _dataTransfers
    .Select(d => d.Value)
    .Where(x => filterByStatus.Contains(x.TransferStatus)).ToList();
```